setup.py

```diff
@@ -25,7 +25,7 @@
       package_dir={ '': 'lib' },
       packages=find_packages('lib'),
       package_data={
-         '': ['module_utils/*.ps1', 'modules/core/windows/*.ps1', 'modules/extras/windows/*.ps1'],
+         '': ['module_utils/*.ps1', 'modules/core/windows/*.ps1', 'modules/extras/windows/*.ps1', 'contrib/README.md', 'contrib/inventory/*'],
```

Comment:
I think we want these listed in MANIFEST.in, not in setup.py's package_data.    I think package_data is for thing that are going to end up in the site-packages/ansible/ directory.  We don't want the contrib/ stuff to end up there as they're more like helper scripts.